src/v/cloud_storage/remote_segment_index.h

```diff
@@ -85,6 +85,14 @@ class offset_index {
     /// returned.
     std::optional<find_result> find_kaf_offset(kafka::offset upper_bound);
 
+    /// Find index entry which is strictly lower than the timestamp
+    ///
+    /// The returned value has kaf_offset less than upper_bound.
```

Comment:
Should this updated to say `timestamp` instead of `kaf_offset`?

Reply:
Yes. Fixed.